plugins/SliceInfoPlugin/SliceInfo.py

```diff
@@ -47,82 +47,78 @@ def messageActionTriggered(self, message_id, action_id):
+                    model_info["bounding_box"]["maximum"]["x"] = node.getBoundingBox().maximum.x
+                    model_info["bounding_box"]["maximum"]["y"] = node.getBoundingBox().maximum.y
+                    model_info["bounding_box"]["maximum"]["z"] = node.getBoundingBox().maximum.z
+                    model_info["transformation"] = str(node.getWorldTransformation().getData())
+
+                    models_info.append(model_info)
+
+        # Bundle the collected data
+        submitted_data = {
+            "processor": platform.processor(),
+            "machine": platform.machine(),
+            "platform": platform.platform(),
+            "settings": global_container_stack.serialize(), # global_container with references on used containers
+            "version": Application.getInstance().getVersion(),
+            "modelhash": "None",
+            "printtime": print_information.currentPrintTime.getDisplayString(),
+            "filament": material_used,
+            "language": Preferences.getInstance().getValue("general/language"),
+            "materials_profiles ": {}
+        }
+        for container in global_container_stack.getContainers():
+            container_id = container.getId()
+            try:
+                container_serialized = container.serialize()
+            except NotImplementedError:
+                Logger.log("w", "Container %s could not be serialized!", container_id)
+                continue
+            if container_serialized:
+                submitted_data["settings_%s" %(container_id)] = container_serialized # This can be anything, eg. INI, JSON, etc.
+            else:
+                Logger.log("i", "No data found in %s to be serialized!", container_id)
 
             # Convert data to bytes
-            submitted_data = urllib.parse.urlencode(submitted_data)
-            binary_data = submitted_data.encode("utf-8")
+        submitted_data = urllib.parse.urlencode(submitted_data)
+        binary_data = submitted_data.encode("utf-8")
 
-            # Submit data
-            try:
-                if Platform.isOSX():
-                    ssl_context = ssl._create_unverified_context()
-                    server_request = urllib.request.urlopen(self.info_url, data = binary_data, timeout = 1, context = ssl_context)
-                else:
-                    server_request = urllib.request.urlopen(self.info_url, data = binary_data, timeout = 1)                Logger.log("i", "Sent anonymous slice info to %s", self.info_url)
-                server_request.close()
-            except Exception as e:
-                Logger.logException("e", "An exception occurred while trying to send slice information")
-        except:
-            # We really can't afford to have a mistake here, as this would break the sending of g-code to a device
-            # (Either saving or directly to a printer). The functionality of the slice data is not *that* important.
```

Comment:
What fieldOfView means is that there might be other exceptions occurring in the future and we basically NEVER want to block the slicing when anything fails in the sending of slicing data.

I am of the opinion that if there is an exception we haven't caught yet, that is a coding error on our side. We should explicitly code this piece to be resilient to internet interruptions and such and test this well, rather than hide our coding errors behind a try: catch block.